src/components/TextField/TextField.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+
+interface ITextFieldProps {
+  customClassNames?: string;
```

Comment:
If you wrap the component in a parent `div`, `customClassNames` could be put to the parent div. I think we might need another property so we can style the input element as well